sdk/openai/azure-openai/samples/chat_completions_quickstart.py

```diff
@@ -0,0 +1,64 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: chat_completions_quickstart.py
+
+DESCRIPTION:
+    This sample demonstrates how to get started with Chat Completions using the Azure OpenAI SDK for Python.
```

Comment:
We should use terminology "official openai SDK for Python" (no Azure)

Reply:
I see the value in the distinction.

In instances like this sample, where we are showcasing only the AOAI usage, is it appropriate to say:
 This sample demonstrates how to get started with Azure OpenAI Chat Completions.

or is it still more encouraged to keep it top-level like:
 This sample demonstrates how to get started with Chat Completions using the official OpenAI SDK for Python

Reply:
I have no strong opinion, I think either works. You could combine them to "This sample demonstrates how to get started with Azure OpenAI Chat Completions using the official OpenAI SDK for Python"?

Reply:
why not both?!